frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -76,6 +76,10 @@ const isPendingPods = (
   (!currentPodCount && !!desiredPodCount);
 
 export const getFailedPods = (pods: ExtPodKind[]): number => {
+  if (!pods.length) {
```

Comment:
This will have the same runtime error as the original bug because `pods` is undefined or null.

```suggestion
  if (!pods?.length) {
```

Reply:
Thanks, I can't read apparently. I'll fix it.